test/raft.test.ts

```diff
@@ -0,0 +1,84 @@
+import { expect } from 'chai'
+import { ethers } from 'hardhat'
+import { Raft } from '../typechain-types'
+import { Event, Wallet } from 'ethers'
+import { waffle } from 'hardhat'
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers'
+
+describe('Raft', async function () {
+    let raftContract: Raft
+    const name = 'Raft'
+    const symbol = 'RAFT'
+    const chainId = 31337
+    const tokenURI = 'blah'
+
+    async function deployContractFixture() {
+        const Raft = await ethers.getContractFactory('Raft')
+        const [owner, addr1] = await ethers.getSigners()
+        const raftContract = await Raft.deploy(owner.address, name, symbol)
+        await raftContract.deployed()
+        return { Raft, raftContract, owner, addr1 }
+    }
+
+    it('should deploy the contract with the right params', async function () {
+        const { raftContract } = await loadFixture(deployContractFixture)
+        const deployedContractName = await raftContract.name()
+        const deployedSymbolName = await raftContract.symbol()
+        expect(deployedContractName).to.equal(name)
+        expect(deployedSymbolName).to.equal(symbol)
+
```

Comment:
Here you should also check the owner of the contract and the paused state as these are being set in the constructor

Reply:
done